taf/updater/updater.py

```diff
@@ -186,6 +186,21 @@ class RepositoryConfig:
         default=False,
         metadata={"docs": "Whether update fails if a warning is raised. Optional."},
     )
+    # JMC: Addition of --no-deps option to disallow updating of the last validated commit if any part of validation is skipped
```

Comment:
Probably don't need any additional comment here because the metadata is descriptive enough. In this case, it's more accurate than this comment